app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  before_action :authenticate_user!
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :photo, :bio, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :photo, :bio, :email, :password, :current_password) }
   end
```

Comment:
- [OPTIONAL] Actually, I'm not able to create any user 🤔 Here is the screenshot error message 👇🏻 And by checking into the rails console there is no user created but since it's working on your side and the previous reviewer did not say any thing about this, I'm going to mark this as an optional required.

![image](https://user-images.githubusercontent.com/79058364/217612465-4934a86b-710b-414c-a72b-4f2bddd1a376.png)